src/main/java/frame/panels/PNLPanel.java

```diff
@@ -1,10 +1,40 @@
 package frame.panels;
 
 import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
 import java.awt.*;
 
 public class PNLPanel extends JPanel{
-    public PNLPanel(){
+    private final String[] currencyTypes;
+    public PNLPanel(String[] currencyTypes){
+        this.currencyTypes = currencyTypes;
+
+        createPNLTab();
+        fillPNLTab();
+    }
+
+    private void createPNLTab() {
         setLayout(new BorderLayout());
     }
+
+    private void fillPNLTab() {
+        String[] columnNames = {"Close",
+                "Date",
```

Comment:
Is this today's date?

Reply:
ignore the pnl stuff - changes are going to be made in separate branch